### PR TITLE
Feature/tile

### PR DESCRIPTION
**Pull request by MrBuggy**

### CODE REVIEW

File: src/es/components/molecules/tile/Tile.js [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+*/
+export default class Tile extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    // Define button type (default secondary)
+    this.buttonType = this.getAttribute('buttonType') || 'secondary';
+    this.modifier = this.getAttribute('modifier') || '';
```
Comment (Weedshaker) [inline src/es/components/molecules/tile/Tile.js:17]: modifier is never used

File: src/es/components/organisms/tileList/TileList.js [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+        }
+      }
+    `
+    return this.fetchTemplate()
+  }
+
+  /**
+   * fetches the template
+   */
+  fetchTemplate () {
```
Comment (Weedshaker) [inline src/es/components/organisms/tileList/TileList.js:209]: this function is useless and can be removed.

**Review by Weedshaker — CHANGES_REQUESTED** [review]
Please, correct the few commented places, else it looks fine.

**Review by Weedshaker — APPROVED** [review]
phone pair approving

**Review by Weedshaker — APPROVED** [review]

**Comment by Weedshaker** [comment]
you can merge it!... after adjusting the add and remove Event Listener stuff

File: src/css/variablesCustom.css [hunk 3]
```diff
@@ -140,4 +140,6 @@
   --search-input-width-big:100%;
   --tag-filter-horizontal-gap: 0.5em;
   --title-border-display: none;
+  --icon-color-blue: #0053A6;
```
[the remark this answers is not in the document]
Comment (MrBuggy) [reply src/css/variablesCustom.css:179]: Finde die Farbe leider nirgends in den Repos. Scheint neu zu sein.

**Review by Weedshaker — APPROVED** [review]

**Pull request merged**